installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc

```diff
@@ -0,0 +1,72 @@
+[id="installing-restricted-networks-aws-installer-provisioned"]
+= Installing a cluster on AWS in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-restricted-networks-aws-installer-provisioned
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on Amazon Web Services (AWS) in a restricted network by creating an internal mirror of the installation release content on an existing Amazon Virtual Private Cloud (VPC).
+
+[id="prerequisites_installing-restricted-networks-aws-installer-provisioned"]
+== Prerequisites
+
+* You xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[created a mirror registry on your mirror host] and obtained the `imageContentSources` data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the mirror host, you can use that computer to complete all installation steps.
+====
+* You created your own VPC in AWS hosting the mirror registry. When installing to a restricted network using installer-provisioned infrastructure, you cannot use the installer-provisioned VPC; your mirror registry must be hosted on a user-provided VPC configured for the cluster to use.
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster.
++
+[IMPORTANT]
+====
+If you have an AWS profile stored on your computer, it must not use a temporary session token that you generated while using a multi-factor authentication device. The cluster continues to use your current AWS credentials to create AWS resources for the entire life of the cluster, so you must use key-based, long-lived credentials. To generate appropriate keys, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users] in the AWS documentation. You can supply the keys when you run the installation program.
```

Comment:
Is this new text? Or is this the same text that we use elsewhere? This text implies that the user is not using manual creds mode, right?

Reply:
@staebler This is a note we provide in all AWS install guides. It originated from doc bug [BZ#1724684](https://bugzilla.redhat.com/show_bug.cgi?id=1724684) a couple years ago. Is it still relevant?

Reply:
We can circle back to this if we want to update it in another PR, since this affects all AWS install guides.

Reply:
Yes, certainly, let's not try to address it in this PR.